notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Cifar10 -&gt; CIFAR-10</p><p> </p><p><a href="https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/cifar10-baseline.html" target="_blank">https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/cifar10-baseline.html</a>  -&gt;</p><p>Learn more about<a href="https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/cifar10-baseline.html " target="_blank">Pytorch Lightning example</a></p><p> </p><p>Vertex to distribute -&gt; Vertex AI to distribute</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>